### PR TITLE
Update moment lib to address regex dos vulnerability

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
Adresses https://nvd.nist.gov/vuln/detail/CVE-2017-18214

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
I performed the update locally, and successfully ran through a quick smoke test of all of the main views and routes.  No bugs or console warnings to report.

**Comment by cbwebdevelopment** [comment]
[Update moment in blip](https://trello.com/c/rJnYAMSK/36-update-moment-in-blip)

File: package.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
     "karma-sourcemap-loader": "0.3.7",
     "karma-webpack": "1.8.0",
     "mocha": "3.0.2",
-    "moment": "2.14.1",
+    "moment": "2.19.3",
```
Comment (krystophv) [inline package.json]: While this satisfies the vulnerability problem, is there a reason we wouldn't just use the latest stable release if we're upgrading it anyway?

Comment (cbwebdevelopment) [reply package.json]: Yeah, that makes more sense :) 

I'll update it.

**Pull request merged**